aten/src/ATen/Context.cpp

```diff
@@ -121,6 +121,14 @@ void Context::setAllowTF32CuDNN(bool b) {
   allow_tf32_cudnn = b;
 }
 
+bool Context::allowTF32Onednn() const {
```

Comment:
```suggestion
bool Context::allowTF32OneDNN() const {
```
Is this better?

Reply:
I perfer we align the style with change from CPU here https://github.com/pytorch/pytorch/pull/133315? @yanbing-j  @guangyey